src/api-types/projects/projects.ts

```diff
@@ -0,0 +1,24 @@
+import { Projects } from '@/types/projects';
+
+export type ProjectsApiModel = {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
We will need to add exception for all files in api-types directory to allow a naming in snake_case notation